### DISCUSSION TITLE
What are the hyperparameters on the molecular datasets leaderboard?

### DISCUSSION THREAD

**Discussion by wzhang2022** [snap-stanford/ogb#184] [General]
Specifically, I'm interested in reproducing the results on the leaderboard that links to the code [here](https://github.com/snap-stanford/ogb/tree/master/examples/graphproppred/mol). Additionally, what search strategy did you use for the hyperparameter search?

**Comment by weihua916** [comment]
The default hyper-parameters are what we have used. We did not really search many hyper-parameters. The primary one is early-stopping and dropout (0 or 0.5, and 0.5 performs better in molecule data).
